test/lib/instruction-parser.spec.ts

```diff
@@ -0,0 +1,106 @@
+import { getPackagesFromRunInstructions } from '../../lib/instruction-parser';
+
+describe('instruction parsers', () => {
+  describe('getPackagesFromRunInstructions', () => {
+    test('supports "apt install"', () => {
+      const instruction = 'RUN apt install curl';
+      const packages = getPackagesFromRunInstructions([instruction]);
+
+      expect(packages).toEqual({ 'curl': { instruction } });
+    });
+
+    test('supports "apt-get install"', () => {
+      const instruction = 'RUN apt-get install curl';
+      const packages = getPackagesFromRunInstructions([instruction]);
+
+      expect(packages).toEqual({ 'curl': { instruction } });
+    });
+
+    test('supports "apk add"', () => {
+      const instruction = 'RUN apk add curl';
+      const packages = getPackagesFromRunInstructions([instruction]);
+
+      expect(packages).toEqual({ 'curl': { instruction } });
+    });
+
+    test('supports "rpm -i"', () => {
+      const instruction = 'RUN rpm -i curl';
+      const packages = getPackagesFromRunInstructions([instruction]);
+
+      expect(packages).toEqual({ 'curl': { instruction } });
+    });
+
+    test('supports "rpm --install"', () => {
+      const instruction = 'RUN rpm --install curl';
+      const packages = getPackagesFromRunInstructions([instruction]);
+
+      expect(packages).toEqual({ 'curl': { instruction } });
+    });
+
+    test('ignores irrelevant flags', () => {
+      const instruction = 'RUN apt-get install -y wget curl';
+      const packages = getPackagesFromRunInstructions([instruction]);
+
+      expect(packages).toEqual({
+        'curl': { instruction },
+        'wget': { instruction },
+      });
+    });
+
+    test('handles multiple spaces', () => {
+      const instruction = 'RUN    apt   install   curl   vim   ';
+      const packages = getPackagesFromRunInstructions([instruction]);
+
+      expect(packages).toEqual({
+        'curl': { instruction },
+        'vim': { instruction },
+      });
+    });
+
+    test('returns multiple packages', () => {
+      const instruction = 'RUN apt install curl wget vim';
+      const packages = getPackagesFromRunInstructions([instruction]);
+
+      expect(packages).toEqual({
+        'curl': { instruction },
+        'wget': { instruction },
+        'vim': { instruction },
+      });
+    });
+
+    test('supports multiple commands using "&&"', () => {
+      const instruction = 'RUN apt install curl && apt install wget';
+      const packages = getPackagesFromRunInstructions([instruction]);
+
+      expect(packages).toEqual({
+        'curl': { instruction },
+        'wget': { instruction },
+      });
+    });
+
+    test('supports multiple commands using ";"', () => {
```

Comment:
Nice ! can we add another test for this instruction:
`const instruction = 'RUN apt install curl; echo "rob"';`

Reply:
Updated the existing case to 

```
const instruction = 'RUN apt install curl; apt install vim; echo "bitwise"';
```

Hope that covers the case you were concerned about